api/auth/center.go

```diff
@@ -236,9 +236,9 @@ func (c *center) checkFormData(r *http.Request) (*Box, error) {
 		return nil, ErrNoAuthorizationHeader
 	}
 
-	submatches := c.postReg.GetSubmatches(MultipartFormValue(r, "x-amz-credential"))
+	submatches := c.postReg.GetSubmatches(MultipartFormValue(r, strings.ToLower(AmzCredential)))
```

Comment:
Not really related to 61264b05f52ed6847d5bdb5f210982ae70424bce

Reply:
Moved to another commit

Reply:
I'd squash it with ee701c4d0762dbc7a1642e93cdaa3b23a9c8a21f.

Reply:
Done